### ISSUE TITLE
Backspace key does not work in developer tools

### ISSUE THREAD

**Issue by GoogleCodeExporter** [noctarius/chromebackspaceonlinux#22]
```
What steps will reproduce the problem?
1. Enable the back space extension
2. Open the developer tools (JS console or any thing else)
3. Attempt to edit text and use the backspace key. Nothing happens. To 
remove a character, one must reposition the cursor and use the 'delete' 
key.

What is the expected output? What do you see instead?
The expected output is for the character to be deleted. Instead nothing 
happens.

What version of the product are you using? On what operating system?
Using 5.0.375.29 beta on Debian Linux (Squeeze) but this happened before 
upgrading to the beta.

```

Original issue reported on code.google.com by `yonkelt...@gmail.com` on 6 May 2010 at 6:00

**Comment by GoogleCodeExporter** [comment]
```
Ok I guess it happens by handling the chrome protocol. I'll limit this to 'new 
tab' and 
'overview' tomorrow.
```

Original comment by `noctariushtc@googlemail.com` on 6 May 2010 at 7:08
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `noctariushtc@googlemail.com` on 6 May 2010 at 7:17

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.49.5
```

Original comment by `noctariushtc@googlemail.com` on 7 May 2010 at 11:17
- Changed state: **Fixed**
